60__qmltypes/qmltypes.csv

```diff
@@ -0,0 +1,82 @@
+nemo-qml-plugin-social-qt5,github.com:sailfishos/nemo-qml-plugin-social,-,-
+nemo-qml-plugin-systemsettings,github.com:sailfishos/nemo-qml-plugin-systemsettings,-,-
+nemo-qml-plugin-thumbnailer-qt5,github.com:sailfishos/nemo-qml-plugin-thumbnailer,-,-
+nemo-qml-plugin-time-qt5,github.com:sailfishos/nemo-qml-plugin-time,-,-
+nemo-transferengine-qt5,github.com:sailfishos/transfer-engine,-,libnemotransferengine-qt5
+pyotherside-qml-plugin-python3-qt5,github.com:sailfishos/pyotherside,packaging/-,-
+qr-filter-qml-plugin,github.com:sailfishos/qr-filter-qml-plugin,-,-
+sailfish-access-control-qt5,github.com:sailfishos/sailfish-access-control,-,sailfish-access-control-qml-plugin
+sailfish-components-accounts-qt5,bitbucket.org:jolla/ui-sailfish-components-accounts,-,-
+sailfish-components-bluetooth-qt5,bitbucket.org:jolla/ui-sailfish-components-bluetooth,-,-
+sailfish-components-contacts-qt5,bitbucket.org:jolla/ui-sailfish-components-contacts,-,-
+sailfish-components-filemanager,bitbucket.org:jolla/ui-sailfish-components-filemanager,-,-
+sailfish-components-gallery-qt5,bitbucket.org:jolla/ui-sailfish-components-gallery,-,-
+sailfish-components-media-qt5,bitbucket.org:jolla/ui-sailfish-components-media,-,-
+sailfish-components-pickers-qt5,bitbucket.org:jolla/ui-sailfish-components-pickers,-,-
+sailfish-components-telephony-qt5,bitbucket.org:jolla/ui-sailfish-components-telephony,-,-
+sailfish-components-timezone-qt5,bitbucket.org:jolla/ui-sailfish-components-timezone,-,-
+sailfish-components-webview-qt5,github.com:sailfishos/sailfish-components-webview,-,"-,-popups,-pickers"
+sailfish-device-encryption,bitbucket.org:jolla/ui-sailfish-device-encryption,-,-settings
+sailfish-mdm,bitbucket.org:jolla/base-sailfish-mdm,-,-
+sailfish-policy,bitbucket.org:jolla/base-sailfish-policy,-,-
+sailfish-secrets,github.com:sailfishos/sailfish-secrets,-,"libsailfishsecretsplugin,libsailfishcryptoplugin"
+sailfishsilica-qt5,bitbucket.org:jolla/ui-sailfish-silica,-,-
+ssu,github.com:sailfishos/ssu,-,-declarative
+"# These are managed by upstream",,,
+#qt5-qtdeclarative,,,
+#qt5-qtdeclarative-import-folderlistmodel,,,
+#qt5-qtdeclarative-import-localstorageplugin,,,
+#qt5-qtdeclarative-import-location,,,
+#qt5-qtdeclarative-import-models2,,,
+#qt5-qtdeclarative-import-multimedia,,,
+#qt5-qtdeclarative-import-particles2,,,
+#qt5-qtdeclarative-import-positioning,,,
+#qt5-qtdeclarative-import-qtquick2plugin,,,
+#qt5-qtdeclarative-import-qttest,,,
+#qt5-qtdeclarative-import-sensors,,,
+#qt5-qtdeclarative-import-settings,,,
+#qt5-qtdeclarative-import-websockets,,,
+#qt5-qtdeclarative-import-window2,,,
+#qt5-qtdeclarative-import-xmllistmodel,,,
+#qt5-qtdeclarative-pim-contacts,,,
+#qt5-qtdeclarative-pim-organizer,,,
+#qt5-qtdeclarative-plugin-qmlstatemachine,,,
+#qt5-qtdeclarative-publishsubscribe,,,
+#qt5-qtdeclarative-qtquick,,,
+#qt5-qtdeclarative-qtquickparticles,,,
+#qt5-qtdeclarative-qtquicktest,,,
+#qt5-qtdeclarative-serviceframework,,,
+#qt5-qtdeclarative-systeminfo,,,
```

Comment:
These last two from qtsystems which we don't even build anymore. No need to have here either?

Reply:
Dropped